app/importer/server/classes/VirtualDataConverter.ts

```diff
@@ -0,0 +1,154 @@
+import { Random } from 'meteor/random';
+
+import { IImportChannel } from '../definitions/IImportChannel';
+import { ImportDataConverter } from './ImportDataConverter';
+import type { IConverterOptions } from './ImportDataConverter';
+
+export class VirtualDataConverter extends ImportDataConverter {
+	protected _objects: Record<string, Array<Record<string, any>>>;
```

Comment:
this could be better typed.. from what I could see, there are three distinct "types": `user` , `channel` and `message` .. also a `Map` seems to fit better.

this way, methods like `getUsersToImport` , `getMessagesToImport` and `getChannelsToImport` could also be better typed, instead of returning `any`